wsi/physicaldisplay.cpp

```diff
@@ -555,6 +557,43 @@ bool PhysicalDisplay::GetDisplayAttribute(uint32_t /*config*/,
   return true;
 }
 
+/* Setting custom resolution instead of preferred as fetched from display */
+bool PhysicalDisplay::SetResolution(const HwcRect<int32_t> &rect) {
+  if ((rect.right - rect.left) && (rect.bottom - rect.top)) {
```

Comment:
u can just use rect.empty() here

Reply:
Default values are set all the time. Also even if left and right or top and bottom are same, the code should handle. So, empty may not work in that case.

Reply:
1,1,1,1 in this case is the expectation to use full resolution than ?

Reply:
Yes, as 1,1,1,1 is not a valid rectangle.